flight/Modules/Sensors/simulated/simsensors.c

```diff
@@ -241,6 +241,7 @@ static void SensorsTask(void *parameters)
 
 static void simulateYasim()
 {
+#if !(defined(_WIN32) || defined(WIN32) || defined(__MINGW32__))
```

Comment:
Yes-- probably should have a different sub-architecture though (like we have common/stm32/stm32f4xx .. could have common/posix/posixwin/ or whatever, with associated defines etc like firmware-arches does